src/txreconciliation.cpp

```diff
@@ -0,0 +1,893 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <txreconciliation.h>
+
+#include <minisketch/include/minisketch.h>
+
+namespace {
+
+/** Current protocol version */
+constexpr uint32_t RECON_VERSION = 1;
+/** Static component of the salt used to compute short txids for transaction reconciliation. */
```

Comment:
A bit of fanciness, but you can insert a new line between each variable. Ease the reading IMHO.

Reply:
We never do this is other places, so I'm leaning towards keeping it with no new lines, unless someone else also asks for it.